From: Debian PHP Maintainers <pkg-php-maint@lists.alioth.debian.org>
Date: Sat, 2 May 2015 10:26:57 +0200
Subject: hack-phpdbg-to-explicitly-link-with-libedit
Updated: 2019-06-04

---
 sapi/phpdbg/config.m4 | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: php-7.2.19/sapi/phpdbg/config.m4
===================================================================
--- php-7.2.19.orig/sapi/phpdbg/config.m4	2019-06-04 07:30:43.411749091 -0400
+++ php-7.2.19/sapi/phpdbg/config.m4	2019-06-04 07:31:25.715947731 -0400
@@ -31,7 +31,9 @@ if test "$BUILD_PHPDBG" = "" && test "$P
   if test "$PHP_PHPDBG_READLINE" = "yes"; then
     if test "$PHP_READLINE" != "no" -o  "$PHP_LIBEDIT" != "no"; then
   	  AC_DEFINE(HAVE_PHPDBG_READLINE, 1, [ ])
-  	  PHPDBG_EXTRA_LIBS="$PHP_READLINE_LIBS"
+#  	  PHPDBG_EXTRA_LIBS="$PHP_READLINE_LIBS"
+# Temporary hack until this gets fixed
+  	  PHPDBG_EXTRA_LIBS="-ledit -ltermcap"
   	  AC_MSG_RESULT([ok])
   	else
   	  AC_MSG_RESULT([readline is not available])
