diff --git a/ext/exif/tests/bug79282.phpt b/ext/exif/tests/bug79282.phpt
index 7b7e3656..df91127c 100644
--- a/ext/exif/tests/bug79282.phpt
+++ b/ext/exif/tests/bug79282.phpt
@@ -7,7 +7,7 @@ var_dump(exif_read_data('data://image/jpeg;base64,/9jhAAlFeGlmAAAg'));
 
 ?>
 --EXPECTF--
-Warning: exif_read_data(): Invalid TIFF alignment marker in %s on line %d
+Warning: exif_read_data(): Missing TIFF alignment marker in %s on line %d
 
 Warning: exif_read_data(): File structure corrupted in %s on line %d
 
